DependencyInjection/FSIDataGridExtension.php

```diff
@@ -43,14 +43,6 @@ public function load(array $configs, ContainerBuilder $container): void
             $container->setParameter('datagrid.twig.themes', $config['twig']['themes']);
         }
 
-        if (true === $container->hasExtension('doctrine')) {
```

Comment:
There are no extensions registered in the container here, because the extensions does not receive them.